From: Roger Shimizu <rogershimizu@gmail.com>
Date: Wed, 13 Dec 2017 00:32:41 +0900
Subject: Change the port to listen from 8388 to 8389

8388 is already taken by default config, so for test suites it's
better to choose a different port.
---
 tests/aes-ctr.json                | 2 +-
 tests/aes-gcm.json                | 2 +-
 tests/aes.json                    | 2 +-
 tests/chacha20-ietf-poly1305.json | 2 +-
 tests/chacha20-ietf.json          | 2 +-
 tests/chacha20.json               | 2 +-
 tests/rc4-md5.json                | 2 +-
 tests/salsa20.json                | 2 +-
 8 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/tests/aes-ctr.json b/tests/aes-ctr.json
index 1f5a1d7..44ba6a2 100644
--- a/tests/aes-ctr.json
+++ b/tests/aes-ctr.json
@@ -1,6 +1,6 @@
 {
     "server":"127.0.0.1",
-    "server_port":8388,
+    "server_port":8389,
     "local_port":1081,
     "password":"aes_password",
     "timeout":60,
diff --git a/tests/aes-gcm.json b/tests/aes-gcm.json
index 271d721..2f74baa 100644
--- a/tests/aes-gcm.json
+++ b/tests/aes-gcm.json
@@ -1,6 +1,6 @@
 {
     "server":"127.0.0.1",
-    "server_port":8388,
+    "server_port":8389,
     "local_port":1081,
     "password":"aes_password",
     "timeout":60,
diff --git a/tests/aes.json b/tests/aes.json
index 2fc29f3..1b3b738 100644
--- a/tests/aes.json
+++ b/tests/aes.json
@@ -1,6 +1,6 @@
 {
     "server":"127.0.0.1",
-    "server_port":8388,
+    "server_port":8389,
     "local_port":1081,
     "password":"aes_password",
     "timeout":60,
diff --git a/tests/chacha20-ietf-poly1305.json b/tests/chacha20-ietf-poly1305.json
index 074dccb..705393d 100644
--- a/tests/chacha20-ietf-poly1305.json
+++ b/tests/chacha20-ietf-poly1305.json
@@ -1,6 +1,6 @@
 {
     "server":"127.0.0.1",
-    "server_port":8388,
+    "server_port":8389,
     "local_port":1081,
     "password":"salsa20_password",
     "timeout":60,
diff --git a/tests/chacha20-ietf.json b/tests/chacha20-ietf.json
index a27987c..2a062a4 100644
--- a/tests/chacha20-ietf.json
+++ b/tests/chacha20-ietf.json
@@ -1,6 +1,6 @@
 {
     "server":"127.0.0.1",
-    "server_port":8388,
+    "server_port":8389,
     "local_port":1081,
     "password":"salsa20_password",
     "timeout":60,
diff --git a/tests/chacha20.json b/tests/chacha20.json
index 6def51b..5c31686 100644
--- a/tests/chacha20.json
+++ b/tests/chacha20.json
@@ -1,6 +1,6 @@
 {
     "server":"127.0.0.1",
-    "server_port":8388,
+    "server_port":8389,
     "local_port":1081,
     "password":"chacha20_password",
     "timeout":60,
diff --git a/tests/rc4-md5.json b/tests/rc4-md5.json
index e69b433..eb08bee 100644
--- a/tests/rc4-md5.json
+++ b/tests/rc4-md5.json
@@ -1,6 +1,6 @@
 {
     "server":"127.0.0.1",
-    "server_port":8388,
+    "server_port":8389,
     "local_port":1081,
     "password":"aes_password",
     "timeout":60,
diff --git a/tests/salsa20.json b/tests/salsa20.json
index a4a664f..3729b46 100644
--- a/tests/salsa20.json
+++ b/tests/salsa20.json
@@ -1,6 +1,6 @@
 {
     "server":"127.0.0.1",
-    "server_port":8388,
+    "server_port":8389,
     "local_port":1081,
     "password":"salsa20_password",
     "timeout":60,
